apps/api/src/app/tenant/usecases/update-tenant/update-tenant.command.ts

```diff
@@ -0,0 +1,20 @@
+import { TenantCustomData } from '@novu/shared';
+import { EnvironmentCommand } from '@novu/application-generic';
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+export class UpdateTenantCommand extends EnvironmentCommand {
+  @IsString()
+  @IsNotEmpty()
+  identifier: string;
+
+  @IsString()
+  @IsOptional()
+  newIdentifier?: string;
```

Comment:
not sure what will be the best name here, but I added the ability to update the identifier from the start because we all know that it is popular to be a `formally 'old company name'` / `previously known as 'old company name'`.
if someone can think of a drawback Speak Now or Forever Hold Your Peace (till the refactor)

Reply:
We don't have to require sending the old `identifier` and the `newIdentifier`, but just one field. 
Also, the validation logic on this field in the use-case should be fixed, because it's possible even to skip the validation if I'll send the same value in both fields.

Reply:
I presented both of them, in order to provide the ability to update identifiers on our end, how can we update the identifier without both of them? perhaps i messed up something?

It is expected to skip the validation because if the user has `identifier : evil-corp` and the new update payload has the `identifier : evil-corp`  which is the same identifier, i don't see a reason to update it or make an unnecessary request to the DB.

Reply:
yeah, I forgot that we are updating with the tenant identifier in the URL and looking for that tenant in db by identifier